MySQLdb/constants/CR.py

```diff
@@ -102,3 +102,55 @@
 AUTH_PLUGIN_ERR = 2061
 MAX_ERROR = 2999
 ERROR_LAST = 2061
+MIN_ERROR = 2000
+UNKNOWN_ERROR = 2000
+SOCKET_CREATE_ERROR = 2001
```

Comment:
Don't duplicate error definitions.

Reply:
`CR_MIN_ERROR` and `CR_MAX_ERROR` needed be excluded in here, and same with `ER_ERROR_FIRST*` and `ER_ERROR_LAST*`.